### ISSUE TITLE
Create a model for shopcart

### ISSUE THREAD

**Issue by nonococoleo** [NYU-DevOps2020-shopcarts/shopcarts#4]
**As a** Developer
**I need** a model file
**So that**  I can define shopcart

**Assumptions:** 
* a shopcart class definition according to its information
* class members may include user id, products in the shopcart, etc. 
* class methods may include add products, remove products, update price, etc.

**Acceptance Criteria:**
```
When I refer to the model's member
Then I can get the detailed information about shopcart
When I call the model's method
Then I can perform change to the shopcart
```

**Comment by iLtc** [comment]
We should move `class methods may include add products, remove products, update price, etc.` to a new story

**Comment by iLtc** [comment]
The model should include `Table Schema`, `serialize`, `deserialize`, and `init_db`.

`create`, `update`, `delete`, and `find` should be done when the actual API is implemented.

**Comment by KHafner** [comment]
Implement model as specified in #12

**Comment by KHafner** [comment]
After #3 is complete, I will check if this passes the test cases. If it does then I will move this to Done

**Comment by KHafner** [comment]
Bug fixes for this are #27
